client/my-sites/exporter/index.jsx

```diff
@@ -36,7 +35,6 @@ function mapStateToProps( state ) {
 		downloadURL: state.siteSettings.exporter.downloadURL,
 		didComplete: getExportingState( state, siteId ) === States.COMPLETE,
 		didFail: getExportingState( state, siteId ) === States.FAILED,
-		showGuidedTransferOptions: config.isEnabled( 'manage/export/guided-transfer' ),
```

Comment:
I'd rather avoid having it in `mapStateToProps` as it's not derived from the Redux store state. While config values aren't going to change throughout the lifecycle of the app, it can be an issue putting non-Redux-state dependent props in `mapStateToProps` as it confuses Redux's update logic. If `state` doesn't change, Redux won't recompute `mapStateToProps`. 

Normally if there are non-Redux-state dependent props which need to be combined with Redux state, they can be passed in at the parent component level then accessed in `mapStateToProps` via the [`ownProps` argument](https://github.com/reactjs/react-redux/blob/master/docs/api.md#arguments). But in this case I don't think we need to make this a prop at all. With this PR the top level exporter component no longer needs to know whether the feature is enabled or not; the Guided Transfer card is shown regardless. Instead, we can just access it where it's needed, inside the component when the button is clicked. Once guided transfer is fully launched and running without issues, the goal is to remove the feature flag check anyway.